### ISSUE TITLE
Aliases and Terminal Profile

### ISSUE THREAD

**Issue by codingbootcampseu** [BringsCoding/bootcamp-schedule#10]
Terminal aliases are used to assign an alternative name to a common action or shell command, i.e. "navigate to your coding folder" or "open a specific file or folder with vs code".

## Alias

The `alias` command can be used to create terminal aliases.

```
> alias sayhello="echo 'hello, you just created a alias'"

> sayhello
hello, you just created a alias
```

> Shell aliases are only available during a terminal session by default.

## Persist Aliases

You can also persist aliases by adding them to your terminal profile file.

Depending on the shell you are using the profile is represented by a different file.

* zsh --> `~/.zshrc`
* bash --> `~/.bash_profile`

If you are not sure what shell you are using please refer [to this particular section within this article](https://www.cyberciti.biz/tips/how-do-i-find-out-what-shell-im-using.html).

> TL;DR: run this command to find your wich shell you are using `printf "My current shell - %s\n" "$SHELL"`

Your terminal profile file is loaded on every start of a new terminal session, which means if you edit the file the changes will be live after you start a new terminal 💡

## Edit Terminal Profile

Run Visual Studio Code to open your profile file like this `/usr/local/bin/code <profileFile>`.

## Todos

- [ ] Add an alias named `code` for Visual Studio Code
- [ ] Add an alias named `coding` that navigates to your coding folder (if you don't have a coding folder yet please create a new folder named `coding` in your home folder)
